Makefile

```diff
@@ -5,6 +5,7 @@
 
 NET ?= 0
 DEBUG ?= 0
+TRANSPARENCY ?= 0
```

Comment:
Why the build tag and variable option? Does the option add much binary size?

Also, for now the feature should probably be activated from the shell rather than be unconditionally enabled with the build tag. I'd remove the build tag and add a command argument to `l` command for now.

In the future we should maybe support embedding a public key in the go-boot binary and settle on a supplementary bt configuration file (or custom UAPI key) to enable this in an authenticated manner.